plugins/modules/vmware_cluster_drs.py

```diff
@@ -214,7 +215,7 @@ def main():
         cluster_name=dict(type='str', required=True),
         datacenter=dict(type='str', required=True, aliases=['datacenter_name']),
         # DRS
-        enable_drs=dict(type='bool', default=False),
+        enable=dict(type='bool', required=True, aliases=['enable_drs']),
```

Comment:
This is a breaking change. What about keeping the default -- False instead? And maybe add a warning saying the key should be set because we will enforce it during the next deprecation cycle.

Reply:
@goneri 
> This is a breaking change.

Sorry, I thought that changing the default from `false` to `true` would be considered a breaking change but making the parameter(s) required wouldn't. I've labeled this PR as work in progress until I've addressed this.

> What about keeping the default -- False instead? And maybe add a warning saying the key should be set because we will enforce it during the next deprecation cycle.

If making `enable_drs/ha/vsan` required is considered a breaking change anyway, I think I'd prefer to _not_ do this but add a warning that the default will change to `true` in the future. I think this would be more consistent with other modules.

Btw: How do we announce breaking changes like this? Ansible version, collection version or a date? I would prefer collection version but I'm not sure. And if so, what would it be... 3?

What do you say?